core/client/fs/src/main/java/alluxio/client/file/AlluxioEvent.java

```diff
@@ -0,0 +1,78 @@
+package alluxio.client.file;
```

Comment:
Why is this in "client"? Shouldn't this be in server?

Also, it probably also shouldn't be in "file" either, since these are really server level events, not file system events.